src/harness/fake_sas.py

```diff
@@ -409,20 +410,31 @@ def do_GET(self):
     self.wfile.write(json.dumps(response))
 
 
-def RunFakeServer(version, is_ecc):
+def RunFakeServer(version, is_ecc, ca_cert_path, verify_crl):
   FakeSasHandler.SetVersion(version)
   if is_ecc:
     assert ssl.HAS_ECDH
   server = HTTPServer(('localhost', PORT), FakeSasHandler)
-  server.socket = ssl.wrap_socket(
-      server.socket,
+  try:
+    with open(ca_cert_path) as file_handle:
+      ca_cert_data = file_handle.read()
+  except IOError:
+    print "%s does not exist in certs path" % ca_cert_path
```

Comment:
Delete `" in certs path"`, because `ca_cert_path` includes `certs/` already.

Reply:
log   message is updated correctly.